src/data.js

```diff
@@ -1,106 +1,219 @@
 'use strict';
 
 /* Program Data
-
   in this file you can declare variables to store important data for your program
   the data can only be primitives, objects or arrays
   do not store dom elements in these variables!!!!
-
   these variables will be imported by your handlers when necessary
     not by your logic
     not by your listeners
 */
 
 export const quizData = {
-  currentQuestionIndex: 0,
```

Comment:
You are again adding modiications in this file :) I assume is not on purpose. Did you use `git add -p` ?

Reply:
Yes I used git add -p but it does not appear to me. Also I did not make any changes to data.js